<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Authorized</class>
 <widget class="QMainWindow" name="Form_Authorized">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Bahnschrift SemiLight</family>
    <pointsize>14</pointsize>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Подключение к БД</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Изображения/Подключение к БД.png</normaloff>Изображения/Подключение к БД.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="InfoaboutConnect">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Авторизация</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_user">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Имя пользователя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_password">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnUser">
       <property name="text">
        <string>Я пользователь</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnconnect">
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Войти</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
